messaging/src/main/java/org/axonframework/eventhandling/AbstractEventProcessor.java

```diff
@@ -143,6 +146,33 @@ protected final void processInUnitOfWork(List<? extends EventMessage<?>> eventMe
         processInUnitOfWork(eventMessages, unitOfWork, ROOT_SEGMENT);
     }
 
+    protected void processInUnitOfWork(Map<EventMessage<?>, ScopeDescriptor> scopeEvents,
+                                       UnitOfWork<EventMessage<?>> unitOfWork) throws Exception {
+        ResultMessage<?> resultMessage = unitOfWork.executeWithResult(() -> {
```

Comment:
Think I know the answer but just verifying. We already allow handler interceptor registration and thus usage on the `DeadlineManager`. That thus means this method would already have called that `DeadlineManager`'s handler interceptors. Would we want an `EventProcessor`'s handler interceptors to also be called on this case? We should definitely describe the expected outcome, at least in javadoc.

Reply:
Makes sense, we might wanna do it.